ehrql/tables/beta/tpp.py

```diff
@@ -85,51 +85,67 @@ def for_patient_on(self, date):
 
 @table
 class apcs(EventFrame):
+    """
+    Admitted patient care spells (APCS) data is provided via the NHS Secondary Uses Service.
+
+    This table gives core details of spells.
+
+    Each row is an in-hospital spell: a period of continuous care within a single trust.
+    """
+
     apcs_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Primary key used to join to the other APCS tables.",
```

Comment:
A user can't join to the other APCS tables, can they? Could we describe this series as an identifier or an ID?